parser/parser.go

```diff
@@ -32,6 +33,10 @@ var (
 	// Version2 is an alias for VersionGROQ1.
 	// Deprecated: Use VersionGROQ1.
 	Version2 Version = 2
+
+	// VersionX is a copy of Version2 with experimental support for GROQ fragments.
+	// Experimental: This is only an experimental version and may change at any time.
+	VersionX Version = 3
```

Comment:
Could this be `-1`? 🤔

Reply:
When I try to do that the tests fail. It doesn't seem to like `//groq:version=-1` or `//groq:version=-1.x.x`

Reply:
ah, sad. how about `0`? 😅